python/ray/services.py

```diff
@@ -492,10 +493,12 @@ def start_ui(redis_address, stdout_file=None, stderr_file=None, cleanup=True):
             port += 1
     new_env = os.environ.copy()
     new_env["REDIS_ADDRESS"] = redis_address
+    token = binascii.hexlify(os.urandom(24))
```

Comment:
Add a comment explaining why we are generating our own token and what the token will be used for.